### PR TITLE
Tuesday May 17th Diagnostic

### PR DESCRIPTION
**Pull request by shireen-bean**
Completed and sumbitting diagnostic.md file

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -35,19 +35,19 @@ Open up this file in Atom, and write your answers below (where indicated).
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+Absolute path. The end destination was arrived at by specifying the entire path from the home directory rather than from changing directories from the current working directory.
```
Comment (berziiii) [inline diagnostic.md:38]: Correct.

File: diagnostic.md [hunk 2]
```diff
@@ -35,19 +35,19 @@ Open up this file in Atom, and write your answers below (where indicated).
...
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+Absolute path because we need to tell the document to point to exactly where the font is hosted.
```
Comment (berziiii) [inline diagnostic.md:44]: Correct.

File: diagnostic.md [hunk 3]
```diff
@@ -35,19 +35,19 @@ Open up this file in Atom, and write your answers below (where indicated).
...
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+Relative path.  Because we would index into it / refer to it from the current working directory of the file. Ex: in an html doc, if the image is in a separate folder than the html doc you would just specify how to get to the image starting from the html doc.
```
Comment (berziiii) [inline diagnostic.md:50]: Correct.

**Comment by berziiii** [comment]
3 correct out of 6 possible. It doesn't appear that you staged and added the new directory `cli-diagnostic` and the file `rhyme.txt` with the string inside that directory. At least it wasn't present in this pull request. Remember that you have to stage all files changed, BUT in separate commits - 1 file per commit.

**Comment by shireen-bean** [comment]
Oh, didn't know that you wanted the cli-diagnostic directory too.
I actually thought it was a text to make sure we were using add .
<filename> specifically rather than just add .

Shireen Kheradpey

shkherad@bu.edu                  (808) 393-7977

**Pull request closed without merging**